bin/gather_pipelines

```diff
@@ -48,22 +49,31 @@ echo "gather_pipelines:$LINENO] inspecting roles rolebindings in tekton-pipeline
 
 # Collect logs and yaml specs for pods referenced by TaskRuns
 NAMESPACES=$(oc get taskruns,eventlisteners --all-namespaces=true --ignore-not-found -o jsonpath='{range .items[*]}{@.metadata.namespace}{"\n"}{end}' | uniq)
-for NAMESPACE in ${NAMESPACES[@]}; do 
+for NAMESPACE in ${NAMESPACES[@]}; do
     # get all the pods names referenced by taskruns
     PODS=$(oc get po -n ${NAMESPACE} -o json | jq -r '.items[].metadata | select (.labels."app.kubernetes.io/managed-by"=="tekton-pipelines" or .labels."app.kubernetes.io/managed-by"=="EventListener")| .name' | tr  '\n' ' ')
     mkdir -p ${LOGS_DIR}/namespaces/${NAMESPACE}/core
     mkdir -p ${LOGS_DIR}/namespaces/${NAMESPACE}/pods
+    mkdir -p ${LOGS_DIR}/namespaces/${NAMESPACE}/events
```

Comment:
To have a cleaner code you can remove this line and modify [this](https://github.com/openshift-pipelines/must-gather/pull/38/files#diff-656aa18ee96c8c2444f69b7bccc55578df634cde1f88c037852df00724c962daR67) to be like:
```
oc adm inspect -n ${NAMESPACE} $(oc get svc,ep,routes,ev -o name -n ${NAMESPACE}) --dest-dir=${LOGS_DIR} > /dev/null
```

Reply:
Yeah that would work, but with above comments we don't need pod and events directory anymore

Reply:
I don't understand why you created [this](https://github.com/openshift-pipelines/must-gather/pull/38/files#diff-656aa18ee96c8c2444f69b7bccc55578df634cde1f88c037852df00724c962daR57) line of code? will that directory ever be populated with events yaml files? By whom?

Reply:
Yeah we don't need this line anymore, I have done the changes locally, just need to push the changes 
waiting for the final comments